### PR TITLE
Fix spelling in yaml.rst

### PR DESCRIPTION
**Pull request by colin-harkins**

### CODE REVIEW

**Review by zmoon — COMMENTED** [review]
Sorry I didn't get to review this before. There are other places where I would add inline code literal (double backtick) markup, e.g. around the 'EPSG:4326', which represents code.

And regarding `.boundary`, the attribute isn't missing, just not set/implemented. Maybe you could adjust the language slightly.

**Comment by colin-harkins** [comment]
The docs build is failing right now because there are a few broken links: https://ruc.noaa.gov/wrf/wrf-chem/ , https://rapidrefresh.noaa.gov/RAPchemEPAsites/ , https://rapidrefresh.noaa.gov/RAPchemEPA/

I imagine it's just temporary so I'll try to rerun that test in a little while.

File: docs/appendix/yaml.rst [hunk 1]
```diff
@@ -143,18 +143,19 @@ models, central longitude and/or central latitude are required, so check the
...
+2) Add a proj4 string or dictionary for ``cartopy.crs.Projection``. Note: If a proj4 
 string or dictionary is used, it must completely define an instance of 
-cartopy.crs.Projection. For example, converting ccrs.PlateCarre() to a proj4 
-string results in {'proj': 'eqc', 'lat_ts': 0, 'lat_0': 0, 'lon_0': 0, 'x_0': 0, 
+cartopy.crs.Projection. For example, converting ccrs.PlateCarree() to a proj4 
+dict results in {'proj': 'eqc', 'lat_ts': 0, 'lat_0': 0, 'lon_0': 0, 'x_0': 0, 
 'y_0': 0, 'ellps': 'WGS84', 'to_meter': 111319.490793274, 'no_defs': None, 
 'type': 'crs'}, but this is not able to completely define an instance of 
-cartopy.crs.Projection due to a the missing .boundary attribute. 
+cartopy.crs.Projection due to the .boundary attribute not yet being implemented
```
Comment (zmoon) [inline docs/appendix/yaml.rst:152]:
```suggestion
``cartopy.crs.Projection`` due to the ``.boundary`` attribute not yet being implemented 
```

File: docs/appendix/yaml.rst [hunk 2]
```diff
@@ -143,18 +143,19 @@ models, central longitude and/or central latitude are required, so check the
...
 string or dictionary is used, it must completely define an instance of 
-cartopy.crs.Projection. For example, converting ccrs.PlateCarre() to a proj4 
-string results in {'proj': 'eqc', 'lat_ts': 0, 'lat_0': 0, 'lon_0': 0, 'x_0': 0, 
+cartopy.crs.Projection. For example, converting ccrs.PlateCarree() to a proj4 
+dict results in {'proj': 'eqc', 'lat_ts': 0, 'lat_0': 0, 'lon_0': 0, 'x_0': 0, 
 'y_0': 0, 'ellps': 'WGS84', 'to_meter': 111319.490793274, 'no_defs': None, 
 'type': 'crs'}, but this is not able to completely define an instance of 
-cartopy.crs.Projection due to a the missing .boundary attribute. 
+cartopy.crs.Projection due to the .boundary attribute not yet being implemented 
+when defining cartopy.crs.Projection from a proj4 string or dict.
```
Comment (zmoon) [inline docs/appendix/yaml.rst:153]:
```suggestion
when defining ``cartopy.crs.Projection`` from a proj4 string or dict. 
```

File: docs/appendix/yaml.rst [hunk 3]
```diff
@@ -143,18 +143,19 @@ models, central longitude and/or central latitude are required, so check the
...
-cartopy.crs.Projection. For example, converting ccrs.PlateCarre() to a proj4 
-string results in {'proj': 'eqc', 'lat_ts': 0, 'lat_0': 0, 'lon_0': 0, 'x_0': 0, 
+cartopy.crs.Projection. For example, converting ccrs.PlateCarree() to a proj4 
+dict results in {'proj': 'eqc', 'lat_ts': 0, 'lat_0': 0, 'lon_0': 0, 'x_0': 0, 
 'y_0': 0, 'ellps': 'WGS84', 'to_meter': 111319.490793274, 'no_defs': None, 
 'type': 'crs'}, but this is not able to completely define an instance of 
-cartopy.crs.Projection due to a the missing .boundary attribute. 
+cartopy.crs.Projection due to the .boundary attribute not yet being implemented 
+when defining cartopy.crs.Projection from a proj4 string or dict. 
 A string such as 'EPSG:4326' will work.(e.g., projection: 'EPSG:4326').
```
Comment (zmoon) [inline docs/appendix/yaml.rst:154]:
```suggestion
A string such as ``'EPSG:4326'`` will work.(e.g., ``projection: 'EPSG:4326'``).
```

**Review by zmoon — COMMENTED** [review]
More code literal markup

**Review by rschwant — APPROVED** [review]
This looks good to me. Let's add it in.

**Comment by zmoon** [comment]
Thanks @colin-harkins !

**Pull request merged**